app/models/condition_dashboard/temperature.rb

```diff
@@ -0,0 +1,23 @@
+module Temperature
+
+  def temp_range(temp_floor)
+    [temp_floor..temp_floor + 9]
+  end
+
+  def condition_trips(temp_floor)
+    conditions = where(max_temperature: temp_range(temp_floor))
+    conditions.joins(:trips).group(:date).count("id")
+  end
+
+  def average_number_of_rides_from_temp(temp_floor)
```

Comment:
this method looks very similar to the method in your `Precipitation` model - is there a way we can organize these models so we don't have to re-write these methods?